TombEngine/Scripting/Internal/TEN/Flow/Level/FlowLevel.h

```diff
@@ -55,4 +56,6 @@ struct Level : public ScriptInterfaceLevel
 	short GetFogMinDistance() const override;
 	short GetFogMaxDistance() const override;
 	short GetFarView() const override;
+	void SetSecrets(short secrets);
```

Comment:
Are these necessary, if you're directly setting secrets field from lua?

Reply:
no. is not necessary for lua. but perhaps it could be necessary in code, / c++ so I had leave this function.